scr/src/main/java/org/apache/felix/scr/impl/ComponentRegistry.java

```diff
@@ -776,9 +758,6 @@ public void run()
     }
 
     public void shutdown() {
```

Comment:
As this function is now empty, I assume it can be removed (or should other cleanup happen here still)?

Reply:
That is my assumption also given the executor will shutdown and cleanup instead.